health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/HCMMyCampaignRowCard.js

```
@@ -236,6 +259,42 @@ const HCMMyCampaignRowCard = ({ key, rowData, tabData }) => {
+              <div>
+                <CardText style={{ margin: 0 }}>{rowData?.additionalDetails?.error}</CardText>
+              </div>,
+            ]}
+            onOverlayClick={() => {
+              setShowErrorPopUp(false);
+            }}
+            onClose={() => {
+              setShowErrorPopUp(false);
+            }}
+            footerChildren={[]}
+          ></PopUp>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)